### PR TITLE
Meta Tags for feeds.

### PR DESCRIPTION
**Pull request by Colin-Lane**
- also removes Experiment check

@andrewconner @stkem

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/PublicPageMetaTags.scala [hunk 1]
```diff
@@ -85,6 +85,15 @@ case class PublicPageMetaFullTags(unsafeTitle: String, url: String, urlPathOnly:
   val lastName = clean(unsafeLastName)
   val fullName = s"$firstName $lastName"
 
+  private def includeFeedTags: String = includeFeedUrl match {
+    case true =>
+      s"""
+         |<link rel="alternate" type="application/rss+xml" title="RSS Feed for $url" href="$url/rss" />
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PublicPageMetaTags.scala]: How about a better title? `<name of library> feed on Kifi` or something

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/PublicPageMetaTags.scala [hunk 2]
```diff
@@ -72,10 +72,10 @@ case class PublicPageMetaPrivateTags(urlPathOnly: String) extends PublicPageMeta
     """.stripMargin
 }
 
-case class PublicPageMetaFullTags(unsafeTitle: String, url: String, urlPathOnly: String, unsafeDescription: String,
-    images: Seq[String], facebookId: Option[String], createdAt: DateTime, updatedAt: DateTime,
-    unsafeFirstName: String, unsafeLastName: String, profileUrl: String, noIndex: Boolean,
-    related: Seq[String]) extends PublicPageMetaTags {
+case class PublicPageMetaFullTags(unsafeTitle: String, url: String, urlPathOnly: String,
+    includeFeedUrl: Boolean = false, unsafeDescription: String, images: Seq[String], facebookId: Option[String],
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PublicPageMetaTags.scala]: Mind removing the default param here? Makes it very easy to skip over this parameter, and we want the API consumer to always think about it.

**Comment by andrewconner** [comment]
Great!

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/PublicPageMetaTags.scala (hunk 2 above)
Comment (Colin-Lane) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PublicPageMetaTags.scala]: yeah, I can remove the default param.

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/PublicPageMetaTags.scala (hunk 1 above)
Comment (Colin-Lane) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PublicPageMetaTags.scala]: I don't think `PublicPageMetaTags` currently takes in a library, or the name of the library as a parameter. I could modify the `includeFeedUrl` to be an `Option[String] or Option[Library]` to pass in the library's name.

**Comment by stkem** [comment]
:+1:

**Comment by stkem** [comment]
If the experiment is not used elsewhere, you can remove it entirely (i.e. from `UserExperiment.scala`)

**Comment by Colin-Lane** [comment]
Yeah, it is not used anywhere anymore, so I will remove it.

**Pull request merged**